frontend/app/component/form/data_entry/voters_and_votes/VotersAndVotesForm.tsx

```diff
@@ -146,20 +144,24 @@ export function VotersAndVotesForm() {
   React.useEffect(() => {
     if (isSaved) {
       window.scrollTo(0, 0);
+      document.getElementById("feedback-title-0")?.focus();
```

Comment:
couldn't we use refs for this as well? 
```
const returnFocusRef = React.useRef<HTMLElement>(null);
...
returnFocusRef.current?.focus();
```

Reply:
Changed in d1bf0ed431654f0ec14291f58a46a924fc3242c0